api/edge_api/identities/views.py

```diff
@@ -0,0 +1,56 @@
+from flag_engine.identities.builders import (
+    build_identity_dict,
+    build_identity_model,
+)
+from rest_framework import status, viewsets
+from rest_framework.exceptions import NotFound
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from environments.identities.models import Identity
+from features.permissions import IdentityFeatureStatePermissions
+
+from .serializers import EdgeIdentityFeatureStateSerializer
+
+
+class EdgeIdentityFeatureStateViewSet(viewsets.ModelViewSet):
```

Comment:
Have we checked that the documentation is all correct?

Reply:
Yep